concrete/tools/conversations/view_ajax.php

```diff
@@ -70,11 +70,38 @@
                 $ml->setItemsPerPage(-1);
             }
 
+            $bID = intval(Request::post('blockID'), 10);
```

Comment:
It's not JavaScript ;)
The `, 10` is superfluous.
Furthermore, `(int) $x` is imho more readable and a bit faster than `intval($x)`

Reply:
Done :)